app/views/inventories/_edit_form.haml

```diff
@@ -0,0 +1,4 @@
+= simple_form_for :inventory, url:edit_inventory_path do |f|
```

Comment:
We often put the `simple_form_for` in the action templates so we can adjust the url there, and change the submit text by action type -- then just the fields go in a shared `_form.haml`.

Reply:
Yes, thank you. I remembered that strategy once I started wanting to use the same fields for the edit action as for the new action, but I haven't pushed that yet.

Reply:
Yep, that will clear up the confusion with the submit labeled "Add" on a file named "edit."

Reply:
✅